lisp/ess-custom.el

```diff
@@ -1821,17 +1821,18 @@ Otherwise, they get their own temporary buffer."
 
 (defvaralias 'ess-eval-visibly-p 'ess-eval-visibly)
 
-(defcustom ess-eval-visibly nil
+(defcustom ess-eval-visibly 'nowait
   "Non-nil means ess-eval- commands display commands in the process buffer.
+If 'nowait (the default), ESS shows input commands in the process
```

Comment:
Please don't say (the default), it's already put in there automatically by Emacs.